docs/source/notes/amp_examples.rst

```diff
@@ -0,0 +1,186 @@
+.. _amp-examples:
+
+Automatic Mixed Precision Examples
+==================================
+
+.. currentmodule:: torch.cuda.amp
+
+.. contents:: :local:
+
+.. _gradient-scaling-examples:
+
+Gradient Scaling
+^^^^^^^^^^^^^^^^
+
+The code snippets below demonstrate recommended use of :class:`torch.cuda.amp.AmpScaler`.
```

Comment:
The following code snippets demonstrate how to use the :class:`torch.cuda.amp.AmpScaler` class to scale the gradients of a mixed-precision network. 

(Is it mixed-precision or mixed precision?)

Reply:
I like mixed precision without the hyphen, because the headline is "Automatic Mixed Precision" not "Automatic Mixed-Precision"

Reply:
Sure.